core/src/calleeFunctions/helpers/oneInch.ts

```diff
@@ -0,0 +1,174 @@
+import { ethers } from 'ethers';
+import { getCalleeAddressByCollateralType } from '../../constants/CALLEES';
+import { getCollateralConfigBySymbol } from '../../constants/COLLATERALS';
+import { getErc20SymbolByAddress } from '../../contracts';
+import { getDecimalChainIdByNetworkType, getNetworkConfigByType } from '../../network';
+import { CollateralConfig } from '../../types';
+import BigNumber from '../../bignumber';
+import { getTokenAddressByNetworkAndSymbol } from '../../tokens';
+import { WAD_NUMBER_OF_DIGITS } from '../../constants/UNITS';
+import { Queue } from 'async-await-queue';
+import memoizee from 'memoizee';
+import { convertETHtoDAI } from '../../fees';
+
+const REQUEST_QUEUE = new Queue(1, 1000);
```

Comment:
- Nit: It's advised to put numbers into self-describing variables
- Let's use a number closer disclosed to their limit? Or did you have any problems with that?
```suggestion
const MAX_DELAY_BETWEEN_REQUESTS_MS = 600;
const REQUEST_QUEUE = new Queue(1, MAX_DELAY_BETWEEN_REQUESTS_MS);
```